<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>819</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PyTrader v0.2</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../PyQt/Qt_Designer/pytrader.ico</normaloff>../PyQt/Qt_Designer/pytrader.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>281</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>수동주문</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Acc">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_order_type">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매수취소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매도취소</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_hoga">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Stock_Code">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Stock_Code_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_num">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_price">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>270</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_order">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>130</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>711</width>
      <height>621</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>잔고 및 보유종목현황</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>691</width>
       <height>101</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>예수금 (d+2)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총매입</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총평가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총수익률 (%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>추정자산</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>691</width>
       <height>421</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>보유량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매입가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수익률 (%)</string>
      </property>
     </column>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>590</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>실시간 조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_look">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>580</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
